apps/nginx.go

```diff
@@ -70,49 +68,7 @@ func (LoggingProcessorNginxAccess) Type() string {
 }
 
 func (p LoggingProcessorNginxAccess) Components(tag string, uid string) []fluentbit.Component {
-	c := confgenerator.LoggingProcessorParseRegex{
-		// Sample line: ::1 - - [26/Aug/2021:16:49:43 +0000] "GET / HTTP/1.1" 200 10701 "-" "curl/7.64.0"
```

Comment:
Do we want to also add a reference to https://docs.nginx.com/nginx/admin-guide/monitoring/logging/#setting-up-the-access-log in `genericAccessLogParser`?

Reply:
Added that alongside the apache docs.